pkg/build/controller/strategy/util.go

```diff
@@ -531,6 +538,45 @@ func setupBuildCAs(build *buildv1.Build, pod *corev1.Pod, additionalCAs map[stri
 		}
 		pod.Spec.Containers = containers
 	}
+
+	if !proxyCAsExist && useProxy {
+		cmSource := &corev1.ConfigMapVolumeSource{
+			LocalObjectReference: corev1.LocalObjectReference{
+				Name: buildutil.GetBuildProxyCAConfigMapName(build),
+			},
+			//TODO the TBD proxy CA injector will update the ConfigMapVolumeSource keyToPath items, right?
```

Comment:
Correct - once the work for the controller is done, we need to explicitly reference the injected key here and give it the new file name `tls-ca-bundle.pem`.

Reply:
If the key name is defined yet in the design doc, I missed it.  Do we know that and the other relevant pieces to scaffold this (even if we comment it out in the short term) @adambkaplan ?

Reply:
As of right now the injected key will be `ca-bundle.crt` - it's buried and easy to miss.

I made a note that the behavior - once finalized, needs to be made abundantly clear:
https://docs.google.com/a/redhat.com/document/d/1y0t0yEOSnKc4abxsjxEQjrFa1AP8iHcGyxlBpqGLO08/edit?disco=AAAADYn95ZA

Reply:
ok I'm going to add some scaffold/prototype code in the commented out section to help capture this